pkg/olm/clusterserviceversion.go

```diff
@@ -3,6 +3,7 @@ package olm
 import (
 	"context"
 	"fmt"
+
```

Comment:
Can you please remove this empty line from the commit?

Reply:
Can you please remove this from commit ?

Reply:
I could not remove this empty line, please stop asking me about it. It was added by goimports tool. when I remove it there is a lint failure on fmt (it's a dead-loop)

Reply:
Just don't run > "Go fmt project". Use >"go fmt file" instead on relevant files that you are changing.

Reply:
go fmt file doesn't help to fix "pkg/clusterversion/clusterversion.go:5: File is not `goimports`-ed (goimports)" issue.
goimports file - helps, bud add an empty line